SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveCampaignState.java

```diff
@@ -0,0 +1,61 @@
+package com.swrve.sdk.messaging;
+
+import com.swrve.sdk.SwrveLogger;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Used internally to save the state of campaigns
+ */
+public class SwrveCampaignState {
+    protected static final String LOG_TAG = "SwrveMessagingSDK";
+
+    // Number of impressions of this campaign. Used to disable the campaign if
+    // it reaches total impressions
+    protected int impressions;
+
+    // Inbox status of the campaign
+    protected SwrveCampaignStatus status;
```

Comment:
I think the actual enums for SwrveCampaignStatus should be inside SwrveCampaignState now.